spec/services/usps_in_person_proofing/transliterator_spec.rb

```diff
@@ -0,0 +1,83 @@
+require 'rails_helper'
+
+RSpec.describe UspsInPersonProofing::Transliterator do
+  describe '#transliterate' do
+    let(:sut) { UspsInPersonProofing::Transliterator.new }
+    context 'baseline functionality' do
+      let(:inputValue) { "\t\n BobИy \t   TЉble?s\r\n" }
+      let(:result) { sut.transliterate(inputValue) }
+      let(:transliteratedResult) { 'Bob?y T?ble?s' }
+
+      let(:inputValue2) { 'Abc Is My Fav Number' }
+      let(:result2) { sut.transliterate(inputValue2) }
+      let(:transliteratedResult2) { inputValue2 }
```

Comment:
in ruby we snake_case variable names
```suggestion
      let(:input_value) { "\t\n BobИy \t   TЉble?s\r\n" }
      let(:result) { sut.transliterate(inputValue) }
      let(:transliterated_result) { 'Bob?y T?ble?s' }

      let(:input_value2) { 'Abc Is My Fav Number' }
      let(:result2) { sut.transliterate(inputValue2) }
      let(:transliterated_result2) { inputValue2 }
```

but also in terms of organization, it seems like the specs mix both result and result2 ... I could consider maybe reorganizing for separate contexts like:


```
let(:result) { transliterator.tranliterate(input) }

context 'with an input that requires transliteration' do
  let(:input) { "\t\n BobИy \t   TЉble?s\r\n" }
  it 'strips whitespace' do
    # etc
  end
end

context 'with an input that needs no transliteration' do
  let(:input) { 'Abc Is My Fav Number' }
  it 'leaves the result as-is' do
    expect(result).to eq(input)
  end
end
```

Reply:
Updated, including with the context split you recommended.